spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/dao/JdbcTaskExecutionDao.java

```diff
@@ -161,6 +163,21 @@
 
 	private DataFieldMaxValueIncrementer taskIncrementer;
 
+	private static final Set<String> validSortColumns = new HashSet<>(10);
+
+	static {
+		validSortColumns.add("TASK_EXECUTION_ID");
```

Comment:
Do we want these checks to be case insenstive.   i.e. lowercase as well as uppercase.

Reply:
Good catch...updated.